lightningd/options.c

```diff
@@ -29,6 +29,8 @@
 #include <unistd.h>
 #include <wire/wire.h>
 
+bool deprecated_apis = true;
+
```

Comment:
Global variable, eek!

Reply:
Yes, but in future this may effect command-line variables too, or deep inside random code.